### PR TITLE
initial implementation for ZPU plugin

### PR DESCRIPTION
**Pull request by tjemg**
For now only supports dissassembly

### CODE REVIEW

File: zpu/zpu.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/* example r_asm plugin by saucec0de at 2016 */
+
+#include <r_asm.h>
+#include <r_lib.h>
+
+static int disassemble (RAsm *a, RAsmOp *op, const ut8 *b, int l) {
+    char arg[10];
```
Comment (radare) [inline zpu/zpu.c]: Use tabs please

File: zpu/zpu.c [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    unsigned char instr = b[0];
```
Comment (radare) [inline zpu/zpu.c]: Use ut8 instead of unsigned char

File: zpu/zpu.c [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        case 0x9: strcpy (op->buf_asm, "NOT");     break;
+        case 0xa: strcpy (op->buf_asm, "FLIP");    break;
+        case 0xb: strcpy (op->buf_asm, "NOP");     break;
+        case 0xc: strcpy (op->buf_asm, "STORE");   break;
+        case 0xd: strcpy (op->buf_asm, "POPSP");   break;
+        case 0xe: strcpy (op->buf_asm, "unknown"); break;
+        case 0xf: strcpy (op->buf_asm, "unknown"); break;
+        default:
+            strcpy (op->buf_asm, "ADDTOP ");
+            sprintf (arg, "%d", instr & 0x0f);
```
Comment (radare) [inline zpu/zpu.c]: This can overflow. Make arg bigger

File: zpu/zpu.c [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        .name = "zpu",
+        .arch = "zpu",
+        .license = "LGPL3",
+        .bits = 32,
+        .desc = "ZPU disassembler",
+        .disassemble = &disassemble,
+};
+
+#ifndef CORELIB
+struct r_lib_struct_t radare_plugin = {
```
Comment (radare) [inline zpu/zpu.c]: Use the typedef name RLibStruct

**Comment by Maijin** [comment]
Please add a PR on https://github.com/radare/radare2-pm too.

**Pull request merged**